doc/changes/changes_1.0.1.md

```diff
@@ -0,0 +1,19 @@
+# Virtual Schema for SAP Hana 1.0.1, released 2021-??-??
+
+Code name: Updated to the latest `virtual-schema-common-jdbc`.
+
+## Refactoring
+
+* #8: Updated to the latest `virtual-schema-common-jdbc`.
+
+## Dependence Updates
+
+### Runtime Dependencies
+
+### Test Dependencies
+
+* Updated ` org.mockito:mockito-junit-jupiter:3.6.0` to `3.7.7`
+
+### Plugin Dependencies
+
+* Updated `com.exasol:project-keeper-maven-plugin:0.4.1` to `0.4.2`
```

Comment:
the removed `org.apache.maven.plugins:maven-failsafe-plugin:3.0.0-M3` dependency is missing